### PR TITLE
fix(example): upgrade with-vuetify 

### LINKED ISSUE #6337: Vuetify needs updating
### What problem does this feature solve?

Vuetify needs to be updated to the latest version. When I try to manually update vuetify, I get a bunch of errors when I try to build the project.

### What does the proposed changes look like?

The Vuetify example should be updated so users can use the latest features of Vuetify.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9713">#c9713</a>)</em></sub></div>

### PR DESCRIPTION
**Pull request by ocBruno**
## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Upgraded the vuetify-example to use the vuetify module created with create-nuxt-app

 I was having some difficulties using the example because of the setup and realized it was outdated. I also thought it would be better to use sass instead of stylus in the example and saw there was an open issue.

Resolves: #6337 

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

I saw there was only one example that had a fixture setup so I wasn't sure if this should be created. If so could someone provide some example cases that I should cover? I'm still  a bit inexperienced with testing. 
Please spot me on any mistakes and I will make sure to fix it if possible! I did make sure to lint my changes and am willing to create the tests if necessary!

### CODE REVIEW

**Review by kevinmarrec — DISMISSED** [review]
Thanks for the upgrade @ocBruno , LGTM.

PS : It could also use https://github.com/nuxt-community/vuetify-module#optionspath instead of `vuetify.theme` in `nuxt.config.js` (to befenit hot reload) but it's still fine.

**Comment by ocBruno** [comment]
Thanks for the feedback, I will make the correction!

File: examples/with-vuetify/vuetify.options.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import colors from 'vuetify/es5/util/colors'
+
+export default {
+  customVariables: ['~/assets/variables.scss'],
```
Comment (kevinmarrec) [inline examples/with-vuetify/vuetify.options.js]: `customVariables` is a top-level moduleOption (https://github.com/nuxt-community/vuetify-module#customvariables) meanwhile `theme` is a framework option.

Which means that `customVariables` should still sit in `nuxt.config.js` :)

**Review by kevinmarrec — CHANGES_REQUESTED** [review]

File: examples/with-vuetify/vuetify.options.js (hunk 1 above)
Comment (ocBruno) [reply examples/with-vuetify/vuetify.options.js]: Thanks for the correction once again!

File: examples/with-vuetify/nuxt.config.js [hunk 2]
```diff
@@ -1,31 +1,63 @@
...
+   */
+  loading: { color: '#fff' },
+  /*
+   ** Global CSS
+   */
+  css: [],
+  /*
+   * Vuetify custom variables
+   */
+  customVariables: ['~/assets/variables.scss'],
```
Comment (kevinmarrec) [inline examples/with-vuetify/nuxt.config.js]: Oops, I meant top-level Vuetify module option i.e. `vuetify.customVariables` (along `optionsPath`)

**Review by kevinmarrec — CHANGES_REQUESTED** [review]

File: examples/with-vuetify/nuxt.config.js (hunk 2 above)
Comment (ocBruno) [reply examples/with-vuetify/nuxt.config.js]: Sorry for the delay I was taking a long needed vacation lol, please notify me if any more changes are necessary or if there are any more possible improvements to this example!

**Comment by kevinmarrec** [comment]
There should have been some misunderstanding, `nuxt.config.js` shoud look like :
```js
{
  vuetify: {
    customVariables: '~/assets/variables.scss',
    optionsPath: 'vuetify.options.js'
  }
}
```
And then `vuetify.options.js` :
```js
{
  theme: {
      dark: true,
      themes: {
        dark: {
          primary: colors.blue.darken2,
          accent: colors.grey.darken3,
          secondary: colors.amber.darken3,
          info: colors.teal.lighten1,
          warning: colors.amber.base,
          error: colors.deepOrange.accent4,
          success: colors.green.accent3
        }
      }
    }
}
```

:smile:

**Comment by ocBruno** [comment]
> There should have been some misunderstanding, `nuxt.config.js` shoud look like :
> 
> ```js
> {
>   vuetify: {
>     customVariables: '~/assets/variables.scss',
>     optionsPath: 'vuetify.options.js'
>   }
> }
> ```
> 
> And then `vuetify.options.js` :
> 
> ```js
> {
>   theme: {
>       dark: true,
>       themes: {
>         dark: {
>           primary: colors.blue.darken2,
>           accent: colors.grey.darken3,
>           secondary: colors.amber.darken3,
>           info: colors.teal.lighten1,
>           warning: colors.amber.base,
>           error: colors.deepOrange.accent4,
>           success: colors.green.accent3
>         }
>       }
>     }
> }
> ```
> 
> 😄

Hey I'm having some problems pushing this commit because of some tests failing and am not sure how to fix this timeout issue.

If anyone has any idea what this might be or how I could debug better, I'm not very familiar with azure pipelines or CircleCI yet I would greatly appreciate it !

Having problems with the following test :
`test('forceExit exits after timeout', () => {
    jest.useFakeTimers()
    const exit = jest.spyOn(process, 'exit').mockImplementation(() => {})
    const stderr = jest.spyOn(process.stderr, 'write').mockImplementation(() => {})

    utils.forceExit('test', 1)
    expect(exit).not.toHaveBeenCalled()
    jest.runAllTimers()

    expect(stderr).toHaveBeenCalledTimes(1)
    expect(stderr).toHaveBeenCalledWith(expect.stringMatching('Nuxt.js will now force exit'))
    expect(exit).toHaveBeenCalledTimes(1)

    stderr.mockRestore()
    exit.mockRestore()
    jest.useRealTimers()
  })`

When I test on local cmd nothing fails but one of the tests does skip.

    `Expected number of calls: 1
    Received number of calls: 15

      198 |     jest.runAllTimers()
      199 | 
    > 200 |     expect(stderr).toHaveBeenCalledTimes(1)
          |                    ^
      201 |     expect(stderr).toHaveBeenCalledWith(expect.stringMatching('Nuxt.js will now force exit'))
      202 |     expect(exit).toHaveBeenCalledTimes(1)
      203 | 

      at Object.test (packages/cli/test/unit/utils.test.js:200:20)

      at Object.test (packages/cli/test/unit/utils.test.js:200:20)`

**Comment by ocBruno** [comment]
Going to close this since I saw that there has been a change to the upstream nuxt config so I'm guessing someone else solved this!

**Pull request closed without merging**